api-docs/rst/dev-guide/concepts.rst

```diff
@@ -476,23 +588,23 @@ To configure a webhook-based policy, you set the ``type`` parameter to
 
 ``change``
      Specifies the number of entities to add or remove, for example "1"
-     implies that 1 server needs to be added. Use to change the number of
-     servers to a specific number. If a positive number is used, servers are
-     added; if a negative number is used, servers are removed.
+     implies that 1 unit needs to be added. Use to change the number of
+     units to a specific number. If a positive number is used, units are
+     added; if a negative number is used, units are removed.
 
 ``changePercent``
      Specifies the change value in per cent. Use to change the percentage of
-     servers relative to the current number of servers. If a positive number
-     is used, servers are added; if a negative number is used, servers are
-     removed. The absolute change in the number of servers is always rounded
-     up. For example, if -X% of the current number of servers translates to
-     -0.5 or -0.25 or -0.75 servers, the actual number of servers that
+     units relative to the current number of units. If a positive number
+     is used, units are added; if a negative number is used, units are
+     removed. The absolute change in the number of units is always rounded
+     up. For example, if -X% of the current number of units translates to
+     -0.5 or -0.25 or -0.75 units, the actual number of units that
      will be shut down is 1.
```

Comment:
A positive value adds units. A negative value removes them. The absolute change in number of units is always rounded up. For example, if X% of the current number of servers translates to 1.2, 1.5, or 1.7, the actual number of units to be shut down is 1.